### ISSUE TITLE
1.3.2 not connecting to public repo

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/existdb-packageservice#2]
the short-lived 1.3.1 worked, the effects of the following error are  the same as eXist-db/packagemanager#6 
```
ERROR (LogFunction.java [writeLog]:193) - (Line: 315 /db/apps/packageservice/modules/packages.xqm) Error while retrieving app packages: It is a dynamic error if evaluation of an expression relies on some part of the dynamic context that has not been assigned a value. variable '$local' is not set. 
2018-04-09 10:50:05,607 [qtp1673844749-58] INFO  (LogFunction.java [writeLog]:199) - (Line: 26 /db/apps/packagemanager/controller.xql) admin/webcomponents-lite.js.mapwebcomponents-lite.js.map/packagemanager 
```
exist 4.0.0
p-manag 4.5.0
p-service 1.3.2
java 8
macOS

**Comment by JoernT** [comment]
working on it

**Comment by JoernT** [comment]
should be fixed in v1.3.3

**Comment by duncdrum** [comment]
indeed, thanks Joern
